src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -195,7 +198,7 @@ def ingest_tiff_udf(
                 raise ValueError
 
         write_context = tiledb.Ctx(config)
-        vfs = tiledb.VFS()
+        vfs = tiledb.VFS(ctx=write_context)
```

Comment:
Pretty similar here the reason we have this `write_context` is basically to allow the user to give a config for the destination path of the ingested file, if for some reason the destination S3 account is different from the one we use for reading the datasource with VFS. Again here the VFS is initialized with the default context of the node after node has the `access_credentials_name`.If `write_context` is None right now we assume that the source and destination S3 paths can be accessed with the credentials_name given in the pod, since the scope_ctx(None) does not create any issue.